src/omnisharp/server.ts

```diff
@@ -68,9 +70,8 @@ const latestVersionFileServerPath = 'releases/versioninfo.txt';
 export class OmniSharpServer {
 
     private static _nextId = 1;
-
     private _readLine: ReadLine;
-    private _disposables: vscode.Disposable[] = [];
+    private _disposables: CompositeDisposable = new CompositeDisposable(Disposable.empty);
```

Comment:
Why?

Reply:
We are trying to remove the dependency on vscode wherever possible. So here we are using the Rx disposables instead.

Reply:
I meant, why is this initialized to a empty composite disposable when line 286 replaces it with a real disposable?

Reply:
Yes, you are right. Removed the empty initialization.